checks/fuzzing.go

```diff
@@ -26,16 +27,25 @@ import (
 // CheckFuzzing is the registered name for Fuzzing.
 const CheckFuzzing = "Fuzzing"
 
+var (
+	ossFuzzRepo    clients.RepoClient
+	errOssFuzzRepo chan error
+	once           sync.Once
+)
+
 //nolint:gochecknoinits
 func init() {
 	registerCheck(CheckFuzzing, Fuzzing)
 }
 
 // Fuzzing runs Fuzzing check.
 func Fuzzing(c *checker.CheckRequest) checker.CheckResult {
-	ossFuzzRepo := githubrepo.CreateGithubRepoClient(c.Ctx, c.Client, c.GraphClient)
-	if err := ossFuzzRepo.InitRepo("google", "oss-fuzz"); err != nil {
-		e := sce.Create(sce.ErrScorecardInternal, fmt.Sprintf("InitRepo: %v", err))
+	once.Do(func() {
+		ossFuzzRepo = githubrepo.CreateGithubRepoClient(c.Ctx, c.Client, c.GraphClient)
+		errOssFuzzRepo <- ossFuzzRepo.InitRepo("google", "oss-fuzz")
```

Comment:
don't we have to `make()` the channel first? otherwise we're writing to a nil channel, which blocks forever.

Reply:
Ah nice catch. Done.